src/i18n/en-us.js

```diff
@@ -316,6 +316,7 @@ export default {
             notYourAddress: "Not your address!",
             primaryAddress: "Primary address",
             subAddress: "Sub-address",
+            subAddressIndex: "Index",
```

Comment:
Since the index always contains a value after it, would it be better to add that in the translation string itself?
I'm not sure if in other languages the position of the value is important.

E.g in english it will be `Index 0` but in another language it might be `0 index`

What are your thoughts?